packages/shared/useDateFormat/index.ts

```diff
@@ -66,7 +66,7 @@ export function formatDate(date: Date, formatStr: string, options: UseDateFormat
     a: () => meridiem(hours, minutes, true),
     aa: () => meridiem(hours, minutes, true, true),
   }
-  return formatStr.replace(REGEX_FORMAT, (match, $1) => $1 || matches[match]?.() || match)
+  return formatStr.replace(REGEX_FORMAT, (match, $1) => $1 || matches[match]?.())
```

Comment:
We should use `??` instead

Reply:
should I change by this way? `matches[match]??()`.

Reply:
I mean 

```ts
 return formatStr.replace(REGEX_FORMAT, (match, $1) => $1 ?? matches[match]?.() ?? match)
```

Reply:
ok! Wait me a moment please, this is my first PR on GitHub.